scss/frost-theme.scss

```diff
@@ -37,6 +37,9 @@ $frost-color-button-tertiary-bg: inherit !global;
 $frost-color-button-tertiary-hover-bg: rgba($frost-color-blue-1, 0.1) !global;
 $frost-color-button-tertiary-active: $frost-color-blue-3 !global;
 
+// Buttons - design
+$frost-color-button-tab-hover-bg: rgba($frost-color-blue-1, 0.05) !global;
```

Comment:
Let's make sure that we need a different hover (0.05) vs. the tertiary (0.1) - ideally we could use a common variable